icu4c/source/i18n/unicode/basictz.h

```diff
@@ -152,6 +152,17 @@ class U_I18N_API BasicTimeZone: public TimeZone {
     virtual void getSimpleRulesNear(UDate date, InitialTimeZoneRule*& initial,
         AnnualTimeZoneRule*& std, AnnualTimeZoneRule*& dst, UErrorCode& status) const;
 
+#ifndef U_HIDE_DRAFT_API
+    /**
+     * Get time zone offsets from local wall time.
+     * @draft ICU 69
+     */
+    virtual void getOffsetFromLocal(
+        UDate date, UTimeZoneLocalOption nonExistingTimeOpt,
+        UTimeZoneLocalOption duplicatedTimeOpt, int32_t& rawOffset,
+        int32_t& dstOffset, UErrorCode& status) const;
+
+#endif /* U_HIDE_DRAFT_API */
```

Comment:
should we also change  kFormer = UCAL_TZ_LOCAL_FORMER, kLatter = UCAL_TZ_LOCAL_LATTER ?

Reply:
done